src/Settings/Flavours/Quickest.hs

```diff
@@ -20,4 +20,4 @@ quickestArgs = sourceArgs SourceArgs
     , hsGhc      = stage0 ? arg "-O" }
 
 quickestRtsWays :: Ways
-quickestRtsWays = pure [vanilla, threaded]
+quickestRtsWays = pure [debug, vanilla, threaded]
```

Comment:
this would be not sufficient, `logging` should be added as well, and more to be added to `libraryWays` -- I think we shouldn't patch like this

Reply:
I'm not sure we want to change `libraryWays` much.

```
$ make show! VALUE=GhcLibWays
GhcLibWays="v  dyn"
```

As such, we'd likely want to retain `libraryWays` to stay `vanilla` only, while replicate the old build systems `GhcRtsWays` which are `GhcRTSWays="l debug thr thr_debug thr_l  dyn debug_dyn thr_dyn thr_debug_dyn l_dyn thr_l_dyn"` for `quickest`.

The reason for adding *only* `debug` was to verify that we did in fact fail due to the mismatch between the rts library ways in the `rts.cabal` file and the ones hadrian sets.

Reply:
And that verification just happened on travis:
```
/home/travis/build/snowleopard/hadrian/ghc/_build/stage1/rts/build/libHSrts-1.0_l.a: copyFile: does not exist (No such file or directory)
```

Reply:
@izgzhen now, should this be fixed in GHC to only build vanilla (and threaded) for quickest? I'd say yes. But we should first align the behavior of hadrian and make based build system, and then fix it proper.

Reply:
indeed